CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.11.1 -- 2024-03-06
+
+Updated the `tree-sitter` dependency to include the required minimal patch version.
```

Comment:
nit: `tree-sitter-graph` dependency

Reply:
No, it was actually the tree-sitter dependency that was changed since the least release. This release is also a patch bump, though. Perhaps that is confusing